.github/workflows/gh-ci.yaml

```diff
@@ -16,7 +16,7 @@ concurrency:
 
 defaults:
   run:
-    shell: bash -l {0}
```

Comment:
Just curious why this is no longer necessary? It's bit magical to me but I often ran into problems without it

Reply:
I thought I had written some answer here - sorry about the delay. The answer is that we're using `setup-python` that happens to be quite sensitive to login shells on macOS (compared to mamba which in my experience is sensitive to non-login shells 😓 ). So you end up needing a pure `bash` shell, otherwise it'll pick up system python in subsequent parts of the workflow.